### DISCUSSION TITLE
Emir's build

### DISCUSSION THREAD

**Discussion by WhoIsMrSentry** [makerforgetech/modular-biped#54] [Show and tell]
While I was thinking about what was missing for the robot, it came to my mind and I started making it. this gripper for micro servo
![image](https://github.com/dmt-labs/modular-biped/assets/134494889/696504db-b2bb-4378-bd24-de0f5ea37293)
![image](https://github.com/dmt-labs/modular-biped/assets/134494889/8d81b86b-ae52-42c6-8cf3-21e06996a689)
![image](https://github.com/dmt-labs/modular-biped/assets/134494889/84adf921-ab96-49a0-8fcc-98a5927a6f1a)
![image](https://github.com/dmt-labs/modular-biped/assets/134494889/573975b7-0533-4238-a55f-3ce8327b30a9)
![image](https://github.com/dmt-labs/modular-biped/assets/134494889/f94f337d-3a47-4bba-bfb1-41a4e77df960)
![image](https://github.com/dmt-labs/modular-biped/assets/134494889/ab39c3e4-ead1-4405-b4ca-0bea5d03e857)

**Comment by WhoIsMrSentry** [comment]
![IMG_20230916_173616_308.jpg](https://github.com/dmt-labs/modular-biped/assets/134494889/261da6b9-7e33-4f4b-ac5f-667a2fc75580)

![IMG_20230916_173620_930.jpg](https://github.com/dmt-labs/modular-biped/assets/134494889/55599ae3-a851-4dd0-9ebe-d8d50f4d9d76)

**Comment by Chompalot** [comment]
An interesting idea, good for holding small things for you. If we can just get it walking it could delivery things also :)
